src/components/Layout/Layout.module.css

```diff
@@ -0,0 +1,25 @@
+.layout-container {
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    /* background-color: #525f7f; */
+}
+
+.content-container {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+}
+
+main {
+    height: 100%;
+    overflow: hidden;
```

Comment:
Maybe try to avoid styling elements directly in css. As far as I saw most of us avoided that by just implementing something like this  `<main className{styles['main']>{children}</main>`  and then just  ```
.main {
}
``` in css instead of `main { }`

Reply:
I figured it could be ok since there should be only one main element, but I will fix it...

Reply:
Yhea dont think its any problem here just for the sake of consistency. Think that everything else looks good.

Reply:
> Yhea dont think its any problem here just for the sake of consistency. Think that everything else looks good.

I think it's fixed